src/components/screen/ScheduleDisplay/ScheduleDisplay.tsx

```diff
@@ -0,0 +1,145 @@
+const divNum = 24 * (60 / sampling)
+const TimeDiv = 86400000 / divNum
+
+interface DateSchedules {
+  [time: number]: number
+}
+
+function findMaxValue(dictionary: DateSchedules): number | undefined {
+  let maxValue: number | undefined
+
+  for (const key in dictionary) {
+    const value = dictionary[key]
+    if (typeof value === 'number' && (maxValue === undefined || value > maxValue)) {
+      maxValue = value
+    }
+  }
+
+  return maxValue
+}
+
+function timeCreate(schedule: Schedule): {
+  dates: number[][]
+  dateSchedules: DateSchedules
+} {
+  let dayEnd: number = 0
+  const dates: number[][] = []
+  let oneDay: number[] = []
+  const dateSchedules: DateSchedules = {}
+  schedule.dates.map((date) => {
+    dayEnd = date + 86400000
+    for (let i = date; i < dayEnd; i = i + TimeDiv) {
+      oneDay.push(i)
+      dateSchedules[i] = 0
+    }
+    dates.push(oneDay)
+    oneDay = []
+  })
+
+  return { dates, dateSchedules }
+}
+
+function countSchedules(
+  dateSchedules: DateSchedules,
+  users: User[] | undefined,
+): DateSchedules {
+  if (!users) {
+    return dateSchedules
+  } else {
+    users.map((user) => {
+      user.availables?.map((available) => {
+        for (let i = available.from; i < available.to; i += TimeDiv) {
+          dateSchedules[i]++
+        }
+      })
+    })
+    return dateSchedules
+  }
+}
+
+function colorArrayCreate(colors: string[], max: number | undefined): string[] {
+  if (!max) {
+    return []
+  }
+  const newColors: string[] = []
+  const divNumber: number = colors.length / (max + 1)
+  newColors.push(colors[0])
+  if (colors.length < max) {
+    for (let i = 1; i < max; i++) {
+      if (Math.floor(i * divNumber) == 0) {
+        newColors.push(colors[1])
+      } else {
+        newColors.push(colors[Math.floor(i * divNumber)])
+      }
+    }
+  } else {
+    for (let i = 1; i <= max; i++) {
+      newColors.push(colors[Math.ceil(i * divNumber)])
+    }
+  }
+  return newColors
+}
+
+const ScheduleDisplay: React.FC<ScheduleDisplayProps> = ({ schedule }) => {
+  const result = timeCreate(schedule)
+  const newDates: number[][] = result.dates
+  const dateSchedule: DateSchedules = result.dateSchedules
+  const newDateSchedules = countSchedules(dateSchedule, schedule.users)
+
+  const max = findMaxValue(newDateSchedules)
+  const bgColorDefine = [
+    'bg-white',
+    'bg-fuchsia-200',
+    'bg-fuchsia-300',
+    'bg-fuchsia-400',
+    'bg-fuchsia-500',
+    'bg-fuchsia-600',
+  ]
+  const colors = colorArrayCreate(bgColorDefine, max)
+  return (
+    <div className='overflow-x-scroll rounded-lg'>
+      <div className='flex items-center p-1 md:p-2'>
+        <div className='flex flex-col'>{}</div>
+        {schedule.dates.map((date, index) => (
+          <>
+            <div className='flex flex-col' key={index}>
+              <p className='select-none border bg-secondary-focus px-5 text-sm tracking-wider text-white md:px-10 md:text-lg'>
+                {format(date, 'M/d')}
```

Comment:
```suggestion
                {format(date, 'M/d/E')}
```
曜日表示した方が見やすいと思ったどちらでも